lhotse/cut/mixed.py

```diff
@@ -846,13 +877,26 @@ def load_audio(self, mixed: bool = True) -> Optional[np.ndarray]:
                 )
             except NonPositiveEnergyError as e:
                 logging.warning(
-                    f'{e} MonoCut with id "{track.cut.id}" will not be mixed in.'
+                    f'{e} DataCut with id "{track.cut.id}" will not be mixed in.'
                 )
 
+        if flattened and not any(track.type == "MultiCut" for track in self.tracks):
+            warnings.warn(
```

Comment:
I don't think that's a serious enough issue to annoy the user with a warning (a lesson learned the hard way for me)